README.adoc

```diff
@@ -60,33 +105,93 @@ Also, you should know that until v1.0.0 We're under development, and these inter
 
 We recommend that you extensively test your module.
 
-See https://decidim.org/modules[Modules page on Decidim.org].
+See {uri-modules}[Modules page on Decidim.org].
 
-=== Authorizations Strategies
+=== 🪪 Permissions
```

Comment:
I would rename this to "Identity verifications and authorizations" to better describe what these features are mostly used for.

Also, I would add another section for "Authentication options" which would briefly describe that you can easily add any authentication provider to Decidim that is provided by OmniAuth:
https://github.com/omniauth/omniauth/wiki/List-of-Strategies

And a link to the existing authentication implementations with a link to the relevant section at the docs.

Reply:
Updated it in 4a3e1f10170784550d226d8662bd0d4829e1b41c and 2e7b40e3de2e7a12c1a2c77b251228a2712de716, feel free to check it out and propose any change.

Reply:
I was also thinking that maybe we could add a link to this section at the Decidim site?
https://decidim.org/modules/#auth

I remembered that was in the documentation site but this is what I meant with the "relevant section at the docs".

Reply:
Oh yes, for sure that makes sense!! I've added it on 640309269c.